metals/src/main/scala/scala/meta/internal/pantsbuild/commands/SwitchCommand.scala

```diff
@@ -54,38 +52,23 @@ object SwitchCommand extends Command[SwitchOptions]("switch") {
         Files.isSymbolicLink(common.bloopDirectory) &&
           Files.readSymbolicLink(common.bloopDirectory) ==
             project.root.bloopRoot.toNIO
+      def runBloopProjects =
+        "to see the list of exported Pants targets run: bloop projects"
       if (isUnchanged) {
         if (isStrict) {
           app.info(
-            s"project '${project.name}' is already active${runBloopProjects(project.common)}"
+            s"project '${project.name}' is already active, $runBloopProjects"
           )
         }
         0
       } else {
-        val isCreated = !Files.isSymbolicLink(project.common.bloopDirectory)
         runSymlink(project, common)
-        val verb = if (isCreated) "created" else "switched to"
-        app.info(
-          s"$verb project '${project.name}'${runBloopProjects(project.common)}"
-        )
+        app.info(s"switched to project '${project.name}', $runBloopProjects")
         0
       }
     }
   }
 
-  private def runBloopProjects(common: SharedOptions): String = {
```

Comment:
This is still needed, we launch IntelliJ immediately before the bloop build is exported. We don't want to print a message saying "run bloop projects" when that command won't list anything

Reply:
I force-pushed this PR without reversion of `Minor polish for Pants integration`